test/integration/forms/form.js

```diff
@@ -129,6 +134,147 @@ context('Patient Form', function() {
       .get('.sidebar')
       .should('not.exist');
 
+    cy
+      .get('.js-history-button')
+      .as('historyBtn')
+      .trigger('mouseover');
+
+    cy
+      .get('.tooltip')
+      .should('contain', 'See Form Response History');
+
+    cy
+      .get('@historyBtn')
+      .click();
+
+    cy
+      .get('@historyBtn')
+      .click();
+
+    cy
+      .get('@historyBtn')
+      .click();
+
+    // Cypress is unable to stub the contentWindow API in Firefox
+    // https://github.com/cypress-io/cypress/issues/136#issuecomment-658574403
+    if (!Cypress.isBrowser('firefox')) {
```

Comment:
can probably combine these two `if`s into the same `if`

Reply:
I think they need to be separate as the `.form__update-region` and `.form__history-region` elements are only present when their corresponding state is enabled. And we need two separate stubs to test the print for each.

Reply:
I'm not sure I'm following.  But I'm suggesting removing the } on line 167 and the `if` on line 170?

Reply:
ah I see the update above.. really the test and the stub could be combined there as well?

Reply:
Oh I see what you're saying 👍